### ISSUE TITLE
Turboshaft not usable

### ISSUE THREAD

**Issue by Tutul-** [ValiZockt/RealFuels-Stock#24]
Still configured to use the LiquidFuel.

I tried to add a cfg for them but I'm not sure why it's not working :-/

```
@PART[*]:HAS[@MODULE[ModuleRoboticServoRotor*]:HAS[@PROPELLANT[LiquidFuel]]]:FOR[z_RealFuels_StockEngines]
{
	@MODULE[ModuleRoboticServoRotor*],*
	{
		@PROPELLANT[LiquidFuel]
		{
			@name = Kerosene
			%resourceFlowMode = STACK_PRIORITY_SEARCH
		}
	}
}
```

**Comment by Tutul-** [comment]
It was my first cfg for the manager, I think I did an error because the propellant is marked as resource 🤔

**Comment by ValentinBischof** [comment]
Fixed by #25, thank you!

**Comment by Tutul-** [comment]
You're welcome :-)
But there aren't any sound or plume, I suppose it's normal ¯\_(ツ)_/¯
